[
    {
        "content": "<p>Hi there. We have 5.13 on RHEL7 installed and have configured Shibboleth according to the documentation. We get an assertion back from our IdP with eppn, sn, uid, mail, and givenName but then we see Dataverse hang and produce the following error.  Any help would be greatly appreciated (I don't understand why port 8080 is being tried and why is it trying to get affiliation from the discofeed in the first place).</p>\n<p>[Payara 5.2022.3] [FINE] [] [edu.harvard.iq.dataverse.authorization.providers.shib.ShibServiceBean] [tid: _ThreadID=104 _ThreadName=http-thread-pool::jk-$<br>\n  Trying to get affiliation from disco feed URL: <a href=\"https://dataverse-stage.lib.berkeley.edu:8080/Shibboleth.sso/DiscoFeed\">https://dataverse-stage.lib.berkeley.edu:8080/Shibboleth.sso/DiscoFeed</a></p>\n<p>Thanks much!<br>\nJay Sundu</p>",
        "id": 412034560,
        "sender_full_name": "Jay Sundu",
        "timestamp": 1704845295
    },
    {
        "content": "<p>Hmm, what do you have <code>dataverse.siteUrl</code> set to?</p>",
        "id": 412129795,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1704889456
    },
    {
        "content": "<p>This: <a href=\"https://guides.dataverse.org/en/5.13/installation/config.html#dataverse-siteurl\">https://guides.dataverse.org/en/5.13/installation/config.html#dataverse-siteurl</a></p>",
        "id": 412130474,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1704889732
    },
    {
        "content": "<p>Port 8080 was indeed in my <code>dataverse.siteUrl</code> and once I removed it the authentication succeeded. Thanks Philip!</p>",
        "id": 412156315,
        "sender_full_name": "Jay Sundu",
        "timestamp": 1704898618
    },
    {
        "content": "<p>Great news!</p>",
        "id": 412159737,
        "sender_full_name": "Philip Durbin ðŸš€",
        "timestamp": 1704899383
    }
]